src/compiler/checker.ts

```diff
@@ -32976,18 +33006,19 @@ namespace ts {
                     const otherKind = node.kind === SyntaxKind.GetAccessor ? SyntaxKind.SetAccessor : SyntaxKind.GetAccessor;
                     const otherAccessor = getDeclarationOfKind<AccessorDeclaration>(getSymbolOfNode(node), otherKind);
                     if (otherAccessor) {

```

Comment:
```suggestion
                    const getter = node.kind === SyntaxKind.GetAccessor ? node : getDeclarationOfKind<AccessorDeclaration>(getSymbolOfNode(node), SyntaxKind.SetAccessor);
                    const setter = node.kind === SyntaxKind.SetAccessor ? node : getDeclarationOfKind<AccessorDeclaration>(getSymbolOfNode(node), SyntaxKind.GetAccessor);
                    if (getter && setter) {
```

and then remove the inner declarations of getter/setter

Reply:
The PR currently changes the order of getter/setter errors so that the getter always gets the error on it; with this suggestion you'd get the same change when checking `this` types: `checkAccessorDeclarationTypesIdentical(getter, setter, ...`

Reply:
Simplified